README.rst

```diff
@@ -91,6 +91,10 @@ To build FogLAMP run the command ``make`` in the top level directory. This will
 Once the *make* has completed you can decide to test FogLAMP from your development environment or you can install it. 
 |br| |br|
 
+
+**NOTE:** *Please note that openssl toolkit is a requirement if we want to use https based REST client and certificate based authentication.*
```

Comment:
Instead of another NOTE, please put as bullet point under the existing NOTE for GCC in this section

Reply:
done